qds_sdk/clusterv2.py

```diff
@@ -0,0 +1,549 @@
+from qds_sdk.qubole import Qubole
+from qds_sdk.resource import Resource
+from qds_sdk.cloud.cloud import Cloud
+from qds_sdk.engine import Engine
+from qds_sdk import util
+import argparse
+import json
+
+def str2bool(v):
+    return v.lower() in ("yes", "true", "t", "1")
+
+class ClusterCmdLine:
+
+    @staticmethod
+    def parsers(action):
+        argparser = argparse.ArgumentParser(
+            prog="qds.py cluster",
+            description="Cluster Operations for Qubole Data Service.")
+        subparsers = argparser.add_subparsers(title="Cluster operations")
+
+        create = subparsers.add_parser("create", help="Create a new cluster")
+        if action == "create":
+            ClusterCmdLine.create_update_clone_parser(create, action="create")
+            create.set_defaults(func=ClusterV2.create)
+
+        update = subparsers.add_parser("update", help="Update the settings of an existing cluster")
+        if action == "update":
+            ClusterCmdLine.create_update_clone_parser(update, action="update")
+            update.set_defaults(func=ClusterV2.update)
+
+        clone = subparsers.add_parser("clone", help="Clone a cluster from an existing one")
+        if action == "clone":
+            ClusterCmdLine.create_update_clone_parser(clone, action="clone")
+            clone.set_defaults(func=ClusterV2.clone)
+
+        return argparser
+
+    @staticmethod
+    def create_update_clone_parser(subparser, action=None):
+
+        # cloud config parser
+        cloud = Cloud.get_cloud()
```

Comment:
as discussed I have removed get cloud function and renamed get_cloud_config to get_cloud which will fetch cloud config parser of cloud instead of cloud name. It has no relation with qubole.py